src/catalog/catalog_set.cpp

```diff
@@ -479,14 +510,20 @@ bool CatalogSet::UseTimestamp(CatalogTransaction transaction, transaction_t time
 	return false;
 }
 
-CatalogEntry &CatalogSet::GetEntryForTransaction(CatalogTransaction transaction, CatalogEntry &current) {
+CatalogEntry &CatalogSet::GetEntryForTransaction(CatalogTransaction transaction, CatalogEntry &current, bool *visible) {
```

Comment:
Can we use `bool &visible` here?

We can create a wrapper method to make this optional, e.g.:

```cpp

CatalogEntry &CatalogSet::GetEntryForTransaction(CatalogTransaction transaction, CatalogEntry &current) {
    bool visible;
    return GetEntryForTransaction(transaction, current, visible);
}

```